calcom/cal.com
# feat: umami app analytics added

This PR is to add Umami analytics in the app store

fixes: #14550 

loom video:
https://www.loom.com/share/686f041377e94d51b95e8074fcccf073?sid=f692ff90-9df9-4bf3-b746-d7cbd49c60bf
## Type of change



- New feature (non-breaking change which adds functionality)


## How should this be tested?

- Add Umami analytics application to an event type.
- Open up the event page and it should automatically track it.

## Mandatory Tasks
